### PR TITLE
Allow to update nextRun in UpdateJob

### PR DESCRIPTION
**Pull request by iwiznia**
Needed for https://github.com/Expensify/Expensify/issues/131401

Added automated tests, more tests in the web PR

### CODE REVIEW

File: plugins/Jobs.cpp [hunk 1]
```diff
@@ -863,8 +864,13 @@ void BedrockJobsCommand::process(SQLite& db) {
         const string& nextRun = result[0][1];
         const string& lastRun = result[0][2];
 
-        // Are we rescheduling?
-        const string& newNextRun = request["repeat"].size() ? _constructNextRunDATETIME(nextRun, lastRun, request["repeat"]) : "";
+        // Passed next run takes priority over the one computed via the repeat feature
+        string newNextRun;
```
Comment (alex-mechler) [inline plugins/Jobs.cpp:868]: We should do some validation on what newNextRun contains, since a malformed nextRun could cause the query to fail.

Comment (iwiznia) [reply plugins/Jobs.cpp:868]: True, but we don't really validate this in any other command that receives it, for example [here](https://github.com/Expensify/Bedrock/blob/5987b1e312250f79bca46d988cd16ef5ca0f1794/plugins/Jobs.cpp#L1053) or [here](https://github.com/Expensify/Bedrock/blob/5987b1e312250f79bca46d988cd16ef5ca0f1794/plugins/Jobs.cpp#L509)

Comment (iwiznia) [reply plugins/Jobs.cpp:868]: Also, not quite true, the query would not fail, it would leave the job with a weird nextRun value

Comment (alex-mechler) [reply plugins/Jobs.cpp:868]: Good old sqlite everything is a string

Given that we aren't validating timestamps anywhere, and that this wouldnt break the query, this is fine

**Pull request merged**